### PR TITLE
feat: add new service openstack_projects

### PR DESCRIPTION
**Pull request by cuadradek**
- Added gen and send scripts for new service openstack_projects to
manage access to OpenStack.

### CODE REVIEW

File: gen/openstack_projects [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+		if($members->{$identifier}) {
+			if ($hasAccess) {
+				push @{$members->{$identifier}->{'projects_access'}}, $projectName;
+			}
+			if ($isManager) {
+				push @{$members->{$identifier}->{'projects_managers'}}, $projectName;
+			}
+			if ($isPersonalProject) {
+				my $status = $data->getMemberAttributeValue(attrName => $A_MEMBER_STATUS, member => $memberId);
```
Comment (metodej) [inline gen/openstack_projects:66]: Here.

File: gen/openstack_projects [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+			my $member = {
+				identifier => $identifier,
+				additional_identifier => \@additionalIdentifier,
+				mail => $mail,
+				projects_access => \@projects_access,
+				projects_managers => \@projects_managers
+			};
+
+			my $status = $data->getMemberAttributeValue(attrName => $A_MEMBER_STATUS, member => $memberId);
```
Comment (metodej) [inline gen/openstack_projects:102]: And here.

**Review by metodej — APPROVED** [review]
Everything seems fine, just a nitpick about unifying argument order...

File: gen/openstack_projects (hunk 1 above)
Comment (cuadradek) [reply gen/openstack_projects]: Done

File: gen/openstack_projects (hunk 2 above)
Comment (cuadradek) [reply gen/openstack_projects]: Done

**Review by Johaney-s — APPROVED** [review]

**Pull request merged**